### PR TITLE
Ash Walkers now has a uniform without suit sensors.

### PR DESCRIPTION
**Pull request by Firecage**
The current gladiator uniforms which Ash Walkers spawns with has suit sensors on them.

This PR gives them their own version of the gladiator uniform which lacks suit sensors.

### CODE REVIEW

**Comment by FantasticFwoosh** [comment]
May i add that the default helmet and jumpsuit offers little to no protection also?

**Comment by WJohn** [comment]
should probably honestly get a unique uniform. maybe something like a goliath hide toga.

**Comment by Firecage** [comment]
@WJohn To be fair though. I wanted to do that. But I suck as spriting anything.

**Comment by ExcessiveUseOfCobblestone** [comment]
send me an image that resembles what you want to see on the lizzies and I'll do my best.

**Comment by FantasticFwoosh** [comment]
Now just imagine its a Goliath leather piece or something (goliath skin can be turned into hairless skin by shaving it by the way, unsure if intentional) in that fashionable tribal waist coat, that shows off the bulked up ashwalker muscles (hiding 6 packs under all that). Ideally spritewise on the shoulders and sides, with arms going through holes and the back covered but front exposed, a replicatable armour

> The last thing a miner should see is a bodied tarzan like physique of raw torque and scales **THRUSTING FURIOUSLY** at them with their spears, illuminated by light & crushing heat of the lava

With a little bit more ornate colosseum mask?

**Comment by ExcessiveUseOfCobblestone** [comment]
aka wonky flintstone outfit, got it.

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -402,6 +402,10 @@
 	can_adjust = 0
 	resistance_flags = 0
 
+/obj/item/clothing/under/gladiator/ash_walker
+	desc = "This gladiator uniform appears to be covered in ash and appears to be fairly dated."
```
Comment (Fox-McCloud) [inline code/modules/clothing/under/miscellaneous.dm]: `This gladiator uniform appears to be covered in ash and to be fairly dated.`

**Comment by KorPhaeron** [comment]
I like the gladiator uniforms

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/clothing/under/miscellaneous.dm]: No, it was dated in a fair manner, before an officer of the law

**Pull request merged**